common/templates/context.go

```diff
@@ -518,3 +518,25 @@ func (s Slice) AppendSlice (slice interface{}) (interface{}, error) {
 
 		return result.Interface(), nil
 }
+
+func (s Slice) StringSlice (flag ...bool) interface{} {
+	strict := false
+	if len(flag) > 0 {
+		strict = flag[0]
+	}
+
+	StringSlice := make([]string, 0, len(s))
+	
+	for _, Sliceval := range s {
+		switch t := Sliceval.(type) {
+			case string :
```

Comment:
Add a case fo fmt.Stringer aswell

Reply:
Added case, made it work for strict = false (default) case because i guess that makes more sense.